tests/main/snap-set/task.yaml

```diff
@@ -46,6 +46,18 @@ execute: |
     fi
     snap get snapctl-hooks foo 2>&1 | MATCH 'snap "snapctl-hooks" has no "foo" configuration option'
 
+    echo "Set foo back"
+    snap set snapctl-hooks command=test-snapctl-get-foo foo=bar
+    if ! obtained=$(snap get snapctl-hooks foo); then
```

Comment:
I have not seen this syntax before. Interesting.

Reply:
Yep, I saw it in some other test the other day and borrowed it ;)